packages/sparseqr/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: sparseqr
+  version: "1.2"
+
+source:
+  # PyPI release has invalid setup.py (missing cffi modules information) so we use github archive instead.
+  sha256: 5611ec439b44f22ea86995c49a229e0a377768520ddc7804cef488a8615655a3
+  url: https://github.com/yig/PySPQR/archive/1a146649773911a1376ee9f60c6948004e11b57c.zip
+
+requirements:
+  run:
+    - pycparser
+    - cffi
+    - numpy
+    - scipy
+    - suitesparse
+    - distutils
+
+build:
+  script: |
+    # FIXME: We are not dealing with poetry build system correctly.
+    #        So for now remove `pyproject.toml` to force pypabuild to use setup.py
+    rm -f pyproject.toml
```

Comment:
I don't understand this actually. We are using `pypa/build` so I thought that it would correctly handle packages that build with poetry. But the resulting `.whl` file was empty.